hub/util/keys.py

```diff
@@ -122,6 +123,26 @@ def get_chunk_id_encoder_key(key: str, commit_id: str) -> str:
     )
 
 
+def get_sequence_encoder_key(key: str, commit_id: str) -> str:
+    if commit_id == FIRST_COMMIT_ID:
+        return "/".join(
+            (
+                key,
+                ENCODED_SEQUENCE_NAMES_FOLDER,
+                ENCODED_CHUNK_NAMES_FILENAME,
+            )
+        )
+    return "/".join(
+        (
+            "versions",
+            commit_id,
+            key,
+            ENCODED_SEQUENCE_NAMES_FOLDER,
+            ENCODED_CHUNK_NAMES_FILENAME,
```

Comment:
let's rename this variable to UNSHARDED_ENCODER_FILENAME as it's getting reused here